compiler-rt/lib/builtins/floatuntitf.c

```diff
@@ -16,6 +16,10 @@
 #include "fp_lib.h"
 #include "int_lib.h"
 
+#define SRC_U128
```

Comment:
I think one way to fix the solaris error would be to lift 
`#if defined(CRT_HAS_TF_MODE)` before this include since it also requires __int128.

Reply:
Or adding a `#ifdef CRT_HAS_128BIT`

Reply:
This works indeed.  `floattitf.c` needs the same patch.  It's unfortunate, however, that `int_to_fp.h`isn't more robust, unconditionally using types whose existance isn't guaranteed everywhere.